commands/host.vegman

```diff
@@ -224,6 +224,53 @@ function cmd_virtualmedia_umount {
   usb-ctrl eject "usb.ms.cli"
 }
 
+# @restrict cmd_config tech,admin
+# @doc cmd_config
+# Host config backup and restore
+function cmd_config {
+  subcommand "$@"
+}
+
+# @sudo cmd_config_default tech,admin
+# @restrict cmd_config_default tech,admin
+# @doc cmd_config_default
+# Restore the factory default Host config
+function cmd_config_default {
+  local yes=""
+  while [[ $# -gt 0 ]]; do
+    case "$1" in
+      -y | --yes) yes="y";;
+      *) abort_badarg "$1";;
+    esac
+    shift
+  done
+
+  echo "Host BIOS settings will be reset to defaults on next boot"
+  if [[ -z "${yes}" ]]; then
+    confirm
+  fi
+  
+  ipmitool chassis bootdev none clear-cmos=yes >/dev/null
+}
+
+# @sudo cmd_config_nodefault tech,admin
+# @restrict cmd_config_nodefault tech,admin
+# @doc cmd_config_nodefault
+# Undo if the factory default Host was set
+function cmd_config_nodefault {
+  echo "Undo Host BIOS setting reset"
+  
+  ipmitool chassis bootdev none >/dev/null
+}
+
+# @sudo cmd_config_status tech,admin
+# @restrict cmd_config_status tech,admin
+# @doc cmd_config_status
+# Show boot settings
+function cmd_config_status {
+    ipmitool chassis bootparam get 5
```

Comment:
Shouldn't you process the answer? How it looks like?

Reply:
Boot parameter version: 1
Boot parameter 5 is valid/unlocked
Boot parameter data: 8000000000
 Boot Flags :
   - Boot Flag Valid
   - Options apply to only next boot
   - BIOS PC Compatible (legacy) boot 
   - Boot Device Selector : No override
   - BIOS verbosity : System Default
   - Console Redirection control : Console redirection occurs per BIOS configuration setting (default)
   - BIOS Mux Control Override : BIOS uses recommended setting of the mux at the end of POST